QuChemPedIAProject/templates/base.html

```diff
@@ -9,8 +8,10 @@
 <html lang="fr">
 <head>
   <!-- Chargement du CSS, bootstrap, javascript et jQuery -->
-
-
+  {% load bootstrap4 %}
```

Comment:
Remove those 4 lines - JQuery, Popper, Boostrap JS and CSS are included in the main bundle